locust/web.py

```diff
@@ -518,6 +514,16 @@ def update_user():
 
             return {}, 201
 
+    @property
+    def login_manager(self):
+        if self.web_login:
+            login_manager = LoginManager()
+            login_manager.init_app(self.app)
+            login_manager.login_view = "login"
+            return login_manager
+
+        raise AttributeError("The login_manager is only available with --web-login.\n")
```

Comment:
We could also quit locust here I was just thinking maybe an AttributeError is a bit nicer